content/blog/rest-api-cucumber-blog/index.md

```diff
@@ -0,0 +1,224 @@
+﻿---
+title: "Automating Rest API's using Cucumber and Java"
+date: "2015-10-30"
+coverImage: ""
```

Comment:
@reddysuren Please provide a coverImage for your blog.

Reply:
@vyasriday , Added Image